README.md

```diff
@@ -1,18 +1,13 @@
 fsharp-cheatsheet
 =================
 
-A typesetted F# Cheatsheet in PDF and HTML formats using F# literate tools
+An F# Cheatsheet in Markup and HTML formats
 
 ## Introduction
 
-This is an attempt to replace the old and obsolete [F# Cheat Sheet](http://www.samskivert.com/code/fsharp/fsharp-cheat-sheet.pdf).
```

Comment:
Replace with putting this or similar at the bottom?
```
## Acknowledgements
- Anh-Dung Phan ([@dungpa](https://github.com/dungpa)) created and maintained the repo for many years before donating it to community ownership under fsprojects
- The original inspiration was the (now obsolete) http://www.samskivert.com/code/fsharp/fsharp-cheat-sheet.pdf
```
Also probably something like

## Contributing

This resource was, is and should remain a community effort. If you're learning F# and something important you didn't learn about isn't covered, you're the very best person to point that out and phrase it in a way that someone new can get running with. If someone posts a PR, the more diversity of reviewers we get, the better. That said, the best PRs are small ones that touch one piece at a time and can hence be reviewed and merged rapidly.

- Primary maintainer: @SpiralOss (backup: @bartelink)

Reply:
Updated in ccfcf750e3165ef4f368bec25f914547d0b59ffe